util-flywayinstaller/src/main/java/com/github/cafapi/util/flywayinstaller/Migrator.java

```diff
@@ -38,6 +38,8 @@ private Migrator()
     {
     }
 
+    // Overloaded method for backward compatibility
```

Comment:
I don't think this comment line is still required with the `@Deprecated` annotation being added now.